compiler/src/iree/compiler/Codegen/Dialect/GPU/Transforms/Passes.td

```diff
@@ -20,6 +20,24 @@ def DistributeMmaToLanesPass :
   ];
 }
 
+def ConcretizeMmaShapesPass :
+    InterfacePass<"iree-gpu-concretize-mma-shapes", "mlir::FunctionOpInterface"> {
+  let summary = "Expands the inner dimensions of iree_gpu.multi_mma ops to match the thread layout";
+  let dependentDialects = [
+    "::mlir::tensor::TensorDialect",
+    "::mlir::iree_compiler::IREE::GPU::IREEGPUDialect",
+    "::mlir::iree_compiler::IREE::VectorExt::IREEVectorExtDialect",
```

Comment:
Why do we need a VectorExt dep here?

Reply:
`getConcreteMFMALayout` generates some `PerDimLayoutAttr` which are unused, but I didn't want to refactor the whole function, so I just added the dependent dialect.

Reply:
gah, that's a pain. Can you add a TODO somewhere to drop that dependency?